packages/aws-cdk-lib/aws-efs/test/efs-file-system.test.ts

```diff
@@ -559,4 +559,62 @@ test('imported file system can not add statements to file system policy', () =>
   expect(importedFileSystemResult).toStrictEqual({
     statementAdded: false,
   });
+});
+
+test('mountTargetOrderInsensitiveLogicalId flag is true', () => {
```

Comment:
Can you update the test to use custom subnets to check that the subnet ID is used as a suffix?

For example:
```
new FileSystem(customVpc, 'EfsFileSystem', {
  vpc: customVpc,
  vpcSubnets: { subnets: [ec2.Subnet.fromSubnetId(stack, 'SubnetExample', 'subnet-1')] },
});
```

Reply:
@lpizzinidev hello, when checking subnetID from created VPC, SubnetID attr tokenised and hard to correlate with template output in this test. 

I think with this PR, we're actually not passing SubnetID attribute to MountTarget's logical IDs, we're passing ${subnet.node.id} which i believe sort of concatenation like `${subnetgroupname}Subnet${index}`. So i'm a bit unsure passing subnet selection to FileSystem Contruct. Did you want me to do this because of some future changes that might have happened in VPC definitions in this repository cause test failure?

Isn't it a good method that checking generated logical ids of mount target from resulting different cfn templates regarding to this feature flag?